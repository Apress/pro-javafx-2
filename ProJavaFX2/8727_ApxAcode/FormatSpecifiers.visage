/*
 * @test
 * @run
 */

var b = true;
var i = 1024;
var n = 3.14;
var str = "Hello, World";

println("Display b in a 15 character field   [{%15b b}].");
println("Display i in a 15 character field   [{%15d i}].");
println("Display n in a 15 character field   [{%15f n}].");
println("Display str in a 15 character field [{%15s str}].");
