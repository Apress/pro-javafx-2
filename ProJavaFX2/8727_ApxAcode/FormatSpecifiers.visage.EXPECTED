Display b in a 15 character field   [           true].
Display i in a 15 character field   [           1024].
Display n in a 15 character field   [       3.140000].
Display str in a 15 character field [   Hello, World].
